cmd/cli-doc/cli-doc.go

```diff
@@ -86,11 +89,74 @@ func referencePrinter(command *cobra.Command, level int) string {
 		"```sh\n"+command.Example+"\n```",
 		command.Long,
 		tableOutput.String(),
-		"```sh\n"+GenerateCLIStructure()+"\n```",
+		"```sh\n"+fmt.Sprint(commandPrinter(command, 0))+"\n```",
 		commandOutput)
 }
 
+func getFlags(flags *pflag.FlagSet) []string {
+	var f []string
+	flags.VisitAll(func(flag *pflag.Flag) {
+		f = append(f, fmt.Sprintf("--%v", flag.Name))
+	})
+	return f
+}
+
+func flattenFlags(flags []string) string {
+	var flagString string
```

Comment:
Can we use https://golang.org/pkg/strings/#Join here?

Reply:
Maybe, I haven't really looked at the code, that command existed before :)